static/src/javascripts/__flow__/types/ophan.js

```diff
@@ -55,17 +55,21 @@ declare type OphanComponentType =
     | 'ACQUISITIONS_THRASHER'
     | 'ACQUISITIONS_EDITORIAL_LINK';
 
-declare type OphanComponent = {|
+declare type OphanComponent = {
     componentType: OphanComponentType,
     id?: string,
-    products: $ReadOnlyArray<OphanProduct>,
+    products?: $ReadOnlyArray<OphanProduct>,
     campaignCode?: string,
-    labels: $ReadOnlyArray<string>
-|};
+    labels?: $ReadOnlyArray<string>
+};
 
-declare type OphanComponentEvent = {|
+declare type OphanComponentEvent = {
```

Comment:
had to make these inexact types for it to work with the object spread syntax below: https://github.com/facebook/flow/issues/2405